app/presenters/result_set_presenter.rb

```diff
@@ -49,8 +51,115 @@ def selected_filter_descriptions
     }.reject(&:empty?)
   end
 
+  def document_in_all_sectors(metadata)
+    metadata[:id] === "sector_business_area" && metadata[:labels].count > 42
+  end
+
+  def documents_by_facits
+    sector_facets = {}
+    all_businesses = {
+      all_businesses: {
+        facet_name: "All Businesses",
+        facet_key: "all_businesses",
+        documents: []
+      }
+    }
+    other_facets = {}
+
+    documents.each do | doc |
+      
+      # return if there is no metadata
+      next unless doc[:document][:metadata].present?
+
+      _filters = @filter_params.without('order','keywords')
+
+      # if no filters are selected then put in all business
+      if _filters.values.length == 0
```

Comment:
@DilwoarH I'm looking at writing tests so that we can production-ise this type of display, is it the case here that if no filters are selected we add all documents to this grouping? If so this can presumably be done outside the per-result loop?